### PR TITLE
chore: add named argument sorted-set example

### PR DESCRIPTION
**Pull request by rishtigupta**

### CODE REVIEW

File: examples/sorted-set-example.php [hunk 1]
```diff
@@ -94,6 +94,18 @@ function printBanner(string $message, LoggerInterface $logger): void
     exit(1);
 }
 
+// fetch sorted set by score (0 to 2) in descending order
+$logger->info("Fetching sorted set $SET_NAME by score in descending order\n");
+$response = $client->sortedSetFetchByScore($CACHE_NAME, $SET_NAME, $minScore = 0, $maxScore = 2, $inclusiveMin = true, $inclusiveMax = true, $order=SORT_DESC);
```
Comment (nand4011) [inline examples/sorted-set-example.php:99]: If you're using named arguments, can't you leave off $inclusiveMin and $inclusiveMax?

Comment (rishtigupta) [reply examples/sorted-set-example.php:99]: I thought we want to show named arguments for those too?

Comment (nand4011) [reply examples/sorted-set-example.php:99]: I was thinking that we would use named arguments to show that you could change sort order without having to specify all of the other arguments, assuming you were using the defaults.

Comment (nand4011) [inline examples/sorted-set-example.php:99]:
```suggestion
$response = $client->sortedSetFetchByScore($CACHE_NAME, $SET_NAME, minScore: 0, maxScore: 2, order: SORT_DESC);
```

**Review by nand4011 — APPROVED** [review]

**Pull request merged**